org.unafrog.funkplot.yml

```diff
@@ -0,0 +1,22 @@
+app-id: org.unafrog.funkplot
```

Comment:
There doesn't seem to be a domain `unaforg.org`. So this app-id doesn't meet the requirements.